README.md

```diff
@@ -12,6 +87,56 @@ The emissions can be modified there, with the only requirement being that `sum(a
+
+The function `_getNewEmissionPerSecond()` and `_getNewDistributionEnd()` defines the new emissions per second and new distribution end for the particular case, which could be modified there to change to modified emissions per second and distribution end.
+
+Similarly you can also run the test via `forge test -vv` which will emit the selector encoded calldata for `setEmissionPerSecond` and `setDistributionEnd` which can be used to make the configuration changes.
+
+## FAQ's:
+
+- Do we need to have and approve the whole liquidity mining reward initially?
+
+  It is generally advisable to have and approve funds for the duration of the next 3 months of the Liquidity Mining Program. However it is the choice of the Emission Admin to do it progressively as well, as the users accrue rewards over time.
+
+- Can we configure mutiple rewards for the same asset?
+
+  Yes, Liquidity Mining could be configured for multiple rewards for the same asset.
+
+- Why do we need to approve funds from the Rewards Vault to the Aave Transfer Strategy contract?
+
+  This is needed so the Transfer Strategy contract can pull the rewards from the Rewards Vault to distribute it to the user when the user claims them.
+  
+- Can I reuse an already deployed transfer strategy?
+    
+    Yes, a transfer strategy could be reused if it has already been deployed for the given network (given that you want the rewards vault, rewards admin and the incentives controller to be the same).
+    
+- If a transfer strategy does not exist, how do I create one?
+
+    The transfer strategy is an immutable contract which determines the logic of the rewards transfer. To create a new pull reward transfer strategy (most     common transfer strategy for liquidity mining) you could use the 
+[PullRewardsTransferStrategy.sol](https://github.com/aave/aave-v3-periphery/blob/master/contracts/rewards/transfer-strategies/PullRewardsTransferStrategy.sol) contract with the following constructor params:
+
+    - `incentivesController`: address of the incentives controller
+    - `rewardsAdmin`: address of the incentives controller for access control
+    - `rewardsVault`: address of the rewards vault containing the funds for the Liquidity Mining program.
+
+    Example to deploy a transfer strategy can be found [here](./scripts/RewardsConfigHelpers.s.sol).
+    
+    _Note: All transfer strategy should inherit from the base contract [TransferStrategyBase.sol](https://github.com/aave/aave-v3-periphery/blob/master/contracts/rewards/transfer-strategies/TransferStrategyBase.sol) and you could also define your own custom transfer                   strategy even with NFT’s as rewards, given that you inherit from the base contract._
+
+- Can we stop the liquidity mining program at any time?
```

Comment:
Better to also indicate how, for example with the distribution end setter and emission